.gitignore

```diff
@@ -16,5 +16,8 @@ bin/
 # emulated balena build will install qemu binaries here
 **/.balena
 
+# ssh keys
+core/id*
+worker/id*
```

Comment:
do we still need this ignore? we stopped generating these keys, correct?

Reply:
ye I'll throw that in the bin